torch/csrc/distributed/rpc/request_callback_no_python.cpp

```diff
@@ -86,7 +87,7 @@ std::shared_ptr<JitFuture> RequestCallbackNoPython::processMessage(
                     ->config());
           }
 
-          processRpcWithErrors(*rpc, messageType, id, retFuture);
+          processRpcWithErrors(*rpc, messageType, id, retFuture, std::move(ctx));
```

Comment:
> [performance-move-const-arg] warning: std::move of the const variable 'ctx' has no effect; remove std::move() or make the variable non-const

this lint failure looks real. Do we need to make this lambda mutable?

Reply:
addressed